bitbots_vision/src/bitbots_vision/vision_modules/field_boundary.py

```diff
@@ -452,10 +470,12 @@ def _compute_field_boundary_points(self):
 
 
 class DownsamplingReversedFieldBoundaryDetector(FieldBoundaryDetector):
+    """
+    The :class:`.DownsamplingReversedFieldBoundaryDetector` uses the reversed detection method and finds the field boundary via scan lines running up from bottom to top.
```

Comment:
This does not explain what this class does differently from the ReversedFieldBoundaryDetector

Reply:
> This does not explain what this class does differently from the ReversedFieldBoundaryDetector

```suggestion
    The :class:`.DownsamplingReversedFieldBoundaryDetector` samples the resolution down
    and uses the reversed detection method and finds the field boundary via scan lines running up from bottom to top.
```